Description: Update manpages Whatis section to proper content
 This patch correct the wathis section to describe properly
 the goal of the tool.
Author: Philippe Thierry
Last-Update: 2017-04-28
--- a/doc/manpages/wcc.8
+++ b/doc/manpages/wcc.8
@@ -1,6 +1,6 @@
 .TH WITCHCRAFT "1" "August 2016" "Witchcraft Compiler Collection (WCC)" "User Commands"
 .SH NAME
-Witchcraft \- manual page for Witchcraft Core Compiler
+Witchcraft \- A tool to manipulate binaries and shared object
 .SH SYNOPSIS
 .B wcc
 [\fIoptions\fR] \fIfile\fR
--- a/doc/manpages/wld.8
+++ b/doc/manpages/wld.8
@@ -1,6 +1,6 @@
 .TH WITCHCRAFT "1" "August 2016" "Witchcraft Compiler Collection (WCC)" "User Commands"
 .SH NAME
-Witchcraft \- manual page for Witchcraft Linker
+Witchcraft \- Transform an ELF executable into a shared library
 .SH SYNOPSIS
 .B wld
 [\fIoptions\fR] \fIfile\fR
